app/views/archive-options.njk

```diff
@@ -7,7 +7,7 @@
   {% include "includes/title-subtitle.njk" %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-one-half">
-    {% if authorisation===false or userRole==='Data Admin'%}
+    {% if authorisation===false or userRole==='Super User'%}
```

Comment:
As above - can we replace the hard coded string with a constant?

Reply:
This one is subject to change for WFP-678 so would rather clean this up in that story (this if statement shouldn't exist basically, is a mismatch from the rbac)